src/bci_build/package/keadhcp/README.md.j2

```diff
@@ -0,0 +1,36 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+ {{ image.description }}
```

Comment:
Consider adding a small description instead of using this variable, otherwise, set a value to `description` in the container definition.

Reply:
added